tools/taskcluster/accept-license.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Is there any reason to have this script, and not just hard-code it in the travis file (like before), if this only gets run on travis? I think an important reason to do this is because the script duplicates the platform and build-tools versions.

If this script still needs to exist, should this move to `tools/travis`?